openshift/app.dc.yaml

```diff
@@ -249,3 +305,27 @@ parameters:
     displayName: Persistent Volume Class name
     required: false
     value: netapp-file-standard
+
+# parameters for logging sidecar
+  - name: LOGGING_HOST_NAME
+    description: The hostname of our FLuentd service running further down the monitoring pipeline
+    displayName: Logging host name
+    required: true
+    value: fluentd-dev.pathfinder.gov.bc.ca
```

Comment:
Remove this default value from template. This must be specified in the pipeline upon template invocation. Add a parameter to the Jenkinsfile where it processes this template.